theme/tokens.ts

```diff
@@ -18,6 +18,8 @@ export const tokens = defineTokens({
     sm: { value: '10px' },
     md: { value: '12px' },
     lg: { value: '16px' },
+    xl: { value: '20px' },
```

Comment:
I added a couple more border radius sizes. We can probably remove these one we determine the style we want. 

Some were set as `50%` / `100%` so I changed these to sensible token sizes. The spinner for example was set to `100%` but the same effect is achieved setting it to `lg`

The QR code was `18px` but I set it to `lg` / `16px` as I thought it looks OK and I didn't want to add tokens not divisible by 4.